Source/WhisperFactory.swift

```diff
@@ -187,6 +216,20 @@ class WhisperFactory: NSObject {
 
 extension WhisperFactory: UINavigationControllerDelegate {
```

Comment:
factory |ˈfakt(ə)rē|
noun (pl. factories)
1 a building or group of buildings where goods are manufactured or assembled chiefly by machine.

We manufacture Whispers, right?